assets/src/edit-story/output/test/page.js

```diff
@@ -532,7 +569,33 @@ describe('Page output', () => {
 
       const content = renderToStaticMarkup(<PageOutput {...props} />);
       expect(content).toContain('Hello, link');
-      expect(content).not.toContain('http://shouldremove.com');
+      expect(content).toContain('http://shouldoutput.com');
+    });
+
+    it('should output a link in page attachment area if page attachment is not set', () => {
+      const props = {
+        id: '123',
+        backgroundColor: { color: { r: 255, g: 255, b: 255 } },
+        page: {
+          id: '123',
+          elements: [
+            BACKGROUND_ELEMENT,
+            {
+              ...TEXT_ELEMENT,
+              link: {
+                url: 'http://shouldoutput.com',
```

Comment:
Hey, [it's available](https://www.namecheap.com/domains/registration/results/?domain=shouldoutput.com)! 🎉